src/pages/actor/ActorDetails.module.css

```diff
@@ -0,0 +1,81 @@
+.pageContainer {
+    background-color: #f9f9f9;
+    min-height: 100vh;
```

Comment:
Yes because i wanted it to have the same size as the html document, as i aimed for changing the color of the background wholly so as aly suggested, i resized the container